shell/platform/fuchsia/flutter/default_session_connection.cc

```diff
@@ -251,7 +282,17 @@ void DefaultSessionConnection::Present() {
                    next_present_session_trace_id_);
   ++next_present_session_trace_id_;
 
-  present_requested_time_ = fml::TimePoint::Now();
+  auto now = fml::TimePoint::Now();
+  present_requested_time_ = now;
+
+  // Flutter is requesting a frame here, so mark it as such.
+  // Inspect updates must run on the inspect dispatcher.
+  async::PostTask(
+      inspect_dispatcher_,
+      [this, now = fml::TimePoint::Now().ToEpochDelta().ToNanoseconds()]() {
+        frames_requested_.Add(1);
```

Comment:
Can we change this to presents_requested? It's a bit more clear, at least for me, since technically the AwaitVsync() call is requesting permission for a frame.

Reply:
Done